README.md

```diff
@@ -7,6 +7,11 @@ These processing steps rely on functionality from several projects. Because they
 ## [Automate Metashape](https://github.com/open-forest-observatory/automate-metashape)
 This project is a wrapper around Agisoft Metashape that allows for reproducible end-to-end automated photogrammetry. The tag is `vx.x.x` and the conda environment should be called `meta`.
 
+# Setup
+Before any scripts can be ran, you must symlink the `_bootstrap.py` script into all working directories (i.e. folders with scripts that are ran directly, not called like other scripts such as those in `utils/` or `config/`). Fortunately the script `symlink_bootstrap.py` exists for that purpose. Simply run that script from the root directory of the project and it will take care of the rest.
```

Comment:
Consider adding a comment that this only needs to be run when additional scripts are added.